### ISSUE TITLE
Flatpak overrides for accessibility

### ISSUE THREAD

**Issue by aaron-gh** [elementary/default-settings#342]
### Problem

By default, many applications that use flatpak are not accessible to a11y technologies, due to the fact that the flatpak sandbox does not talk to the a11y bus.

### Proposal

I propose we make the following flatpak environment settings global defaults:
``bash
flatpak override --user --talk-name="org.a11y.bus"
flatpak override --user --env="QT_LINUX_ACCESSIBILITY_ALWAYS=true"
flatpak override --user --env="GTK_PATH=/app/lib/gtkmodules"
``
